pdns/syncres.cc

```diff
@@ -536,6 +536,10 @@ int SyncRes::doResolve(const DNSName &qname, const QType &qtype, vector<DNSRecor
 
           boost::optional<Netmask> nm;
           res=asyncresolveWrapper(remoteIP, d_doDNSSEC, qname, qtype.getCode(), false, false, &d_now, nm, &lwr);
+
+          d_totUsec += lwr.d_usec;
+          accountAuthLatency(lwr.d_usec, remoteIP.sin4.sin_family);
```

Comment:
I think this incidentally solves  #5207 and #5187?

Reply:
I don't think it does?

Reply:
ah, this has indeed nothing to do with the speed tracking per-ns